store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -468,6 +483,13 @@ private void buildTableSchema(Field[] fields, TableSchemaBuilder tableSchemaBuil
     }
   }
 
+  private void isDataTypeSupportedForCarbonWriter(DataType dataType) {
```

Comment:
Change this method name as these data type is not supported for complex data type's children column